stable/restore/values.yaml

```diff
@@ -56,15 +56,25 @@ admin:
   #   secret: nuodb-client-pem
   #   key: client.pem
 
-database:
-  # name
-  # NuoDB Database name.  must consist of lowercase alphanumeric
-  # characters '[a-z0-9]+'
-  name:
-
 restore:
+
+  ## Provide a name in place of prometheus-operator for `app:` labels
```

Comment:
"prometheus-operator" ?

Reply:
Yup - an ***old*** copy-n-paste error.

The "prometheus-operator" string is in the stable/database/values.yaml file as well.

Presumably, the original database chart was copied from the public prometheus operator...

I'll fix it here - we should have a separate PR to fix it in database.